guides/release-notes.md

```diff
@@ -7,6 +7,25 @@ We love hearing your feedback! You can help us improve the docs by reporting any
 ![Was this helpful, yes/no buttons](../media/feedback.png)
 
 ---
+### New API: Dashboard Elements
+
+`Wix Editor Elements ($w)`
+
+The new [Dashboard Elements](https://www.wix.com/velo/reference/$w/dashboardbutton) API allows you to create customized, interactive dashboards for your users.
```

Comment:
> allows you to create customized, interactive dashboards for your users.

This is a little too vague and marketing-y. This also isn't the main benefit of these APIs.

The main benefit, as I understand it, is that the elements are designed to fit in with the dashboard look-and-feel, and this release exposes the APIs for them.